### DISCUSSION TITLE
Pluto TV More Star Trek

### DISCUSSION THREAD

**Discussion by nabeelr** [iptv-org/community#948] [Show and tell]
### Channel ID

MoreStarTrek.us

### Stream URL (optional)

https://service-stitcher.clusters.pluto.tv/v1/stitch/embed/hls/channel/634dacf51d90320007fcd5fa/master.m3u8?advertisingId=channel&appName=rokuchannel&appVersion=1.0&bmodel=bm1&channel_id=channel&content=channel&content_rating=ROKU_ADS_CONTENT_RATING&content_type=livefeed&coppa=false&deviceDNT=1&deviceId=channel&deviceMake=rokuChannel&deviceModel=web&deviceType=rokuChannel&deviceVersion=1.0&embedPartner=rokuChannel&genre=ROKU_ADS_CONTENT_GENRE&is_lat=1&platform=web&rdid=channel&studio_id=viacom&tags=ROKU_CONTENT_TAGS

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://service-stitcher.clusters.pluto.tv/v1/stitch/embed/hls/channel/634dacf51d90320007fcd5fa/master.m3u8?advertisingId=channel&appName=rokuchannel&appVersion=1.0&bmodel=bm1&channel_id=channel&content=channel&content_rating=ROKU_ADS_CONTENT_RATING&content_type=livefeed&coppa=false&deviceDNT=1&deviceId=channel&deviceMake=rokuChannel&deviceModel=web&deviceType=rokuChannel&deviceVersion=1.0&embedPartner=rokuChannel&genre=ROKU_ADS_CONTENT_GENRE&is_lat=1&platform=web&rdid=channel&studio_id=viacom&tags=ROKU_CONTENT_TAGS (offline)
<img width="835" alt="image" src="https://github.com/iptv-org/iptv/assets/7253922/c77c6658-c3bc-4387-b600-6d4f7ebfcac6">

**Comment by GlamoramaAttack** [comment]
EPG is included in:

https://raw.githubusercontent.com/matthuisman/i.mjh.nz/master/PlutoTV/us.xml.gz

as

```
<channel id="634dacf51d90320007fcd5fa">
<display-name>More Star Trek</display-name>
 <icon src="https://images.pluto.tv/channels/634dacf51d90320007fcd5fa/colorLogoPNG.png"/>
</channel>
```

Alternative stream for 'More Star Trek' is on Samsung:

> http://stitcher-ipv4.pluto.tv/v1/stitch/embed/hls/channel/634dacf51d90320007fcd5fa/master.m3u8?deviceType=samsung-tvplus&deviceMake=samsung&deviceModel=samsung&deviceVersion=unknown&appVersion=unknown&deviceLat=0&deviceLon=0&deviceDNT=%7BTARGETOPT%7D&deviceId=%7BPSID%7D&advertisingId=%7BPSID%7D&us_privacy=1YNY&samsung_app_domain=%7BAPP_DOMAIN%7D&samsung_app_name=%7BAPP_NAME%7D&profileLimit=&profileFloor=&embedPartner=samsung-tvplus

**Comment by nabeelr** [comment]
Why is it listed as offline? It's currently online, and 24/7.

Did someone mistake the title card that gets shown during commercials, as the stream being offline?

**Reply by freearhey** [reply]
Last time I checked the stream was not working, but it is indeed working again at the moment.

Created a new request: https://github.com/iptv-org/iptv/issues/14149

**Reply by nabeelr** [reply]
Hmm, weird. I wonder if it's geoblocked? Or maybe they just simply had an outage.
